functions.php

```diff
@@ -34,34 +38,71 @@ public function __construct() {
 		add_action( 'init', array( $this, 'register_taxonomies' ) );
 		add_action( 'cmb2_admin_init', array( $this, 'register_header_metabox' ) );
 		add_action( 'pre_get_posts', array( $this, 'tags_support_query' ) );
+		add_action( 'admin_init', array( $this, 'add_copyright_text' ) );
 
 		// Default actions are in: https://core.trac.wordpress.org/browser/tags/4.5.3/src/wp-includes/default-filters.php#L0
 		remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
 		remove_action( 'wp_head', 'wp_generator' );
 		remove_action( 'wp_print_styles', 'print_emoji_styles' );
-		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_parent_styles') );
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_parent_styles' ) );
 		parent::__construct();
 	}
 
+	/**
+	 * Show copyright text field.
+	 *
+	 * @param array $args
+	 */
+	public function cp_show_settings( $args ) {
+		$copyright = esc_attr( get_option( 'copyright', '' ) );
+
+		printf(
+			'<input type="text" name="copyright" class="regular-text" value="%1$s" id="%2$s" />',
+			esc_attr( $copyright ),
+			esc_attr( $args['label_for'] )
+		);
+	}
+
+	public function add_copyright_text() {
+		add_settings_section(
+			'copyrighttext_id',
+			'',
+			'',
+			'general'
+		);
+
+		// Register a callback.
+		register_setting(
+			'general',
+			'copyright',
+			'trim'
+		);
+		// Register the field for the "copyright" section.
+		add_settings_field(
+			'copyright',
+			'Copyright Text',
+			array( $this, 'cp_show_settings' ),
+			'general',
+			'copyrighttext_id',
+			array(
+				'label_for' => 'copyrighttext_id',
+			)
+		);
+	}
 
 	public function enqueue_parent_styles() {
 		wp_enqueue_style( 'bootstrap', 'https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css', array(), '4.0.0-alpha.6' );
 		wp_enqueue_style( 'parent-style', get_template_directory_uri() . '/style.css' );
-		wp_register_script('jquery', 'https://code.jquery.com/jquery-3.2.1.min.js', array(), null, true);
-		wp_enqueue_script( 'main', get_template_directory_uri() . '/assets/js/main.js', array('jquery'), null, true );
+		wp_register_script( 'jquery', 'https://code.jquery.com/jquery-3.2.1.min.js', array(), null, true );
+		wp_enqueue_script( 'main', get_template_directory_uri() . '/assets/js/main.js', array( 'jquery' ), null, true );
 	}
 
-	/**
-	 *
-	 */
 	public function register_post_types() {
 		//this is where you can register custom post types
 
 	}
 
-	/**
-	 * To register taxonomies for page.
-	 */
+	// To register taxonomies for page.
```

Comment:
Function docs should use a multiline comment style. So:

```
/**
 * Register taxonomies for page.
 */
```

Reply:
Yes done with it.